--- downloads/Dwarf Fortress/data/init/init.txt	2013-02-01 13:54:08.000000000 -0600
+++ lnp_gfx/[16x16] Ironhand 0.73.4/data/init/init.txt	2013-08-21 06:38:11.000000000 -0500
@@ -8,11 +8,11 @@
 
 Change this to turn sound off.
 
-[SOUND:YES]
+[SOUND:NO]
 
 The sound volume runs from 0 (off) to 255 (full).  You can set the volume from the ESC options menu as well.
 
-[VOLUME:0]
+[VOLUME:255]
 
 Change this to skip the wonderful intro movies.
 
@@ -74,7 +74,7 @@
 
 Switches truetype on whenever the vertical tile size is this many pixels or larger.  You may also use YES/NO to force it on/off.  Turning it off may provide a minor speed boost (using font small enough that it doesn't activate won't).
 
-[TRUETYPE:YES]
+[TRUETYPE:NO]
 
 Change this to YES to keep the DF window on top of your other windows.
 
@@ -82,7 +82,7 @@
 
 Change this to YES if you want to see an FPS counter at the top left.
 
-[FPS:NO]
+[FPS:YES]
 
 Use this to set the maximum frame rate during play.  The movies are always capped at 100.  A frame in this case is not graphical but related to the movement speed of a creature.  A healthy, unencumbered dwarven peasant takes one step every 10 frames.
 
@@ -114,7 +114,7 @@
 Use this to set how fast the game zooms.  The default corresponds to
 increasing grid size by 10 units each time you zoom.
 
-[ZOOM_SPEED:10]
+[ZOOM_SPEED:5]
 
 Change this to NO if you don't want to have the mouse involved at all.
 
@@ -139,7 +139,7 @@
 
 This controls the number of milliseconds between macro instructions.
 
-[MACRO_MS:15]
+[MACRO_MS:0]
 
 This controls the number of milliseconds that must pass before input works again after the view recenters on an event in dwarf mode.
 
@@ -147,4 +147,4 @@
 
 Change this to NO if you want to leave save uncompressed (you might want to do this if you are experience save corruption).
 
-[COMPRESSED_SAVES:YES]
\ No newline at end of file
+[COMPRESSED_SAVES:YES]
