--- downloads/CLA_graphic_set_v15-STANDALONE/CLA/data/init/init.txt	2012-07-29 16:33:06.000000000 -0500
+++ lnp_gfx/[16x16] CLA v15/data/init/init.txt	2013-08-21 05:40:13.000000000 -0500
@@ -8,7 +8,7 @@
 
 Change this to turn sound off.
 
-[SOUND:YES]
+[SOUND:NO]
 
 The sound volume runs from 0 (off) to 255 (full).  You can set the volume from the ESC options menu as well.
 
@@ -16,7 +16,7 @@
 
 Change this to skip the wonderful intro movies.
 
-[INTRO:YES]
+[INTRO:NO]
 
 This lets you set the starting windowed/fullscreen setting.  Can be YES, NO or PROMPT.
 
@@ -25,8 +25,8 @@
 This is the size and font for windowed mode.  Changing to the 800x600 font might make you happier.
 If set below 256x256 it specifies the grid size instead, with a minimum of 80x25.
 
-[WINDOWEDX:80]
-[WINDOWEDY:25]
+[WINDOWEDX:0]
+[WINDOWEDY:0]
 [FONT:CLA.png]
 
 You may disable window resizing if you wish.
@@ -114,7 +114,7 @@
 Use this to set how fast the game zooms.  The default corresponds to
 increasing grid size by 10 units each time you zoom.
 
-[ZOOM_SPEED:10]
+[ZOOM_SPEED:5]
 
 Change this to NO if you don't want to have the mouse involved at all.
 
@@ -139,7 +139,7 @@
 
 This controls the number of milliseconds between macro instructions.
 
-[MACRO_MS:15]
+[MACRO_MS:0]
 
 This controls the number of milliseconds that must pass before input works again after the view recenters on an event in dwarf mode.
 
@@ -147,4 +147,4 @@
 
 Change this to NO if you want to leave save uncompressed (you might want to do this if you are experience save corruption).
 
-[COMPRESSED_SAVES:YES]
\ No newline at end of file
+[COMPRESSED_SAVES:YES]
